src/rpc/methods/gas.rs

```diff
@@ -195,19 +210,31 @@ where
         .unwrap_or_default();
 
     let ts = data.mpool.cur_tipset.lock().clone();
+    // Pretend that the message is signed. This has an influence on the gas
+    // cost. We obviously can't generate a valid signature. Instead, we just
+    // fill the signature with zeros. The validity is not checked.
+    let mut chain_msg = match from_a.protocol() {
+        Protocol::Secp256k1 => ChainMessage::Signed(SignedMessage::new_unchecked(
+            msg,
+            Signature::new_secp256k1(vec![0; 65]),
```

Comment:
Let's use a constant from `ref-fvm`; `SECP_SIG_LEN`.

Reply:
Do you think I can use `SECP_SIG_LEN` for the delegated signature as well?

Reply:
I don't think delegated addresses _must_ to use `secp256k1` and thus their length - https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0055.md#delegated-signature-type

That said, delegated signatures are used, to my understanding, for Ethereum messages, which also use `secp256k1`, so we should be safe assuming it's that as well. I'd  use the constant and add a comment.